src/main/java/com/somemore/community/repository/board/CommunityBoardRepositoryImpl.java

```diff
@@ -52,6 +52,18 @@ public List<CommunityBoardView> findByWriterId(UUID writerId) {
                 .fetch();
     }
 
+    @Override
+    public boolean existsById(Long id) {
+        QCommunityBoard communityBoard = QCommunityBoard.communityBoard;
+
+        return queryFactory
+                .selectOne()
+                .from(communityBoard)
+                .where(communityBoard.id.eq(id)
+                        .and(communityBoard.deleted.eq(false)))
+                .fetchFirst() != null;
+    }
```

Comment:
```suggestion
@Override
public boolean existsById(Long id) {
    return communityBoardJpaRepository.existsByIdAndDeletedFalse(id);
}
```
1. 위 코드로 변경하는 것이 어떤지 여쭤보고 싶습니다.
2. dsl로 존재 체크하는 fetchExists도 deprecated 된건지도 여쭤보고 싶습니다.

Reply:
existsByIdAndDeletedFalse가 너무 긴 거 같아서 querydsl로 했었는데 이렇게 보니 jpa로 하는게 훨씬 간결할 거 같네요! 수정하겠습니다.


찾아보니 fetchCount() 메서드가 deprecated 되어서 fetchFirst() 또는 limit(1)을 사용하여 존재 여부를 확인하는 방식이 권장되고 있다고 하네요!

Reply:
감사합니다. 조건이 1개가 아니라서 찝찝하긴 한데, id and deleted는 괜찮지 않을까 싶어서 제안드렸습니다.